### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by sh1zicus** [sh1zicus/dots-hyprland-old#35] [Ideas]
Share ideas for new features

**Comment by PharmaRacist** [comment]
Hello !
i hope ur doin well G 🖤
i recently thought of making side bar tab like the github one using yt api to make music app would be great if we can implement such thing and also add music control widget in right sidebar

**Reply by sh1zicus** [reply]
> Hello !
> i hope ur doin well G 🖤
> i recently thought of making side bar tab like the github one using yt api to make music app would be great if we can implement such thing and also add music control widget in right sidebar

I recently tried to add a lofi player but it didn’t work, it took too long to load music, I used lowfi, maybe I’ll try something else tomorrow

**Reply by PharmaRacist** [reply]
maybe think also of a lyrics module even tho its not that practical but i think it will be aesthetic
